### DISCUSSION TITLE
Default time column setting for time series

### DISCUSSION THREAD

**Discussion by oplehto** [grafana/clickhouse-datasource#780] [Ideas]
Having a default for the time series time column would be useful and reduce mistakes. We typically have the time column uniformly named for all of our metrics but some tables also have secondary time columns. Sometimes users accidentally use the secondary columns instead of the primary time column. Having a default would likely eliminate these mistakes and also make the queries a bit more user-friendly. 

The admin would define the column in the datasource settings (similar to how its done with Traces and Logs) and that column would be used as the time column for macros like `$__timeFilter` if no parameter is given. A user could override this if necessary, adding the time column parameter. This is the current functionality so it wouldn't break existing queries

**Comment by SpencerTorres** [comment]
This would be useful. I haven't thought of this since most of the datasets I test/work with use different time column names.

Long term it might be useful to allow configuring default options for everything, but for now adding one more just for the time column isn't a bad idea
